context-extraction/src/main/java/de/atb/context/persistence/context/ContextRepository.java

```diff
@@ -52,7 +51,7 @@
  */
 @Setter
 @Getter
-public final class ContextRepository extends RepositorySDB<ContextContainer> implements IContextRepository {
+public final class ContextRepository extends RepositoryTDB<ContextContainer> implements IContextRepository {
```

Comment:
There is quite some code duplication in the `executeSparql...` and `getLastContexts...` methods.
Maybe this could be cleaned up?

Reply:
partly done. Maybe some further improvements could be made in future